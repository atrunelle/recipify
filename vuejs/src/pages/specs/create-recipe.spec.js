import { mount } from '@vue/test-utils';
import CreateRecipe from '@/pages/create-recipe';

describe('Component: create recipe', () => {
  let component;

  beforeEach(() => {
    component = mount(CreateRecipe, {
      stubs: ['make-recipe'],
    });
  });

  it('should match snapshot', () => {
    expect(component.html()).toMatchSnapshot();
  });
});
