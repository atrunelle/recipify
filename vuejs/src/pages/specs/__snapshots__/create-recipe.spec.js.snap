// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: create recipe should match snapshot 1`] = `
<div class="layout row wrap">
    <!---->
</div>
`;
